### PR TITLE
Installation using ansible readme

### PR DESCRIPTION
**Pull request by ravisaxena23**
<!--  Thanks for sending a pull request!  Here are some tips for you:

**What type of PR is this?**

> /kind documentation

**What this PR does / why we need it**:

**Test Report Added?**:
> /kind NOT-TESTED

**Special notes for your reviewer**:
I have made documentation for installation of soda using ansible

### CODE REVIEW

**Comment by anvithks** [comment]
Hi @ravisaxena23 .
Thanks for the PR.
Please edit the description and add a meaningful description with some details:
What this PR does?
Why is it required?
What are the changes (if any) that you are making and bugs or issues that are being fixed.

File: content/try-opensds/installation.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ SODA Foundation Projects may be installed, deployed and tested using multiple to
 
 The recommended method of installation is using ansible in Ubuntu 16.04. Detailed steps are provided below.
 
-* [Local Cluster installation using ansible](https://github.com/sodafoundation/api/wiki/SODA-Projects-Cluster-Installation-through-Ansible)
+* [Local Cluster installation using ansible](installation-using-ansible.md)
```
Comment (anvithks) [inline content/try-opensds/installation.md:15]: This link doesn't work.  
I think you need to change it to
`[Local Cluster installation using ansible](try-opensds/installation-using-ansible)`
Note the missing `.md`.

File: content/try-opensds/installation-using-ansible.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+title: Installation Using ansible
+description: "This is a  guide for users and new contributors to install Local Cluster using ansible."
+weight: 30
+---
+This document describes how to install SODA projects local cluster, including Hotpot (API, Controller, Dock), Gelato, Telemetry, Orchestration, and Dashboard components.
+
+`Hereafter, Hotpot refers to API, Controller and Dock projects`
+
+### Pre-config (Ubuntu 16.04)
+All the installation work is tested on `Ubuntu 16.04`, please make sure you have installed the right one. Also `root` user is REQUIRED before the installation work starts.
```
Comment (anvithks) [inline content/try-opensds/installation-using-ansible.md:11]: I think we are going to add official support for `Ubuntu 18.04` as well. 
@kumarashit please confirm.

File: content/try-opensds/installation-using-ansible.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+<br /> Install docker-compose:
+
+```bash
+curl -L "https://github.com/docker/compose/releases/download/1.23.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+chmod +x /usr/local/bin/docker-compose
+```
+<br /> Install golang
+
+```bash
+wget https://storage.googleapis.com/golang/go1.12.1.linux-amd64.tar.gz
```
Comment (anvithks) [inline content/try-opensds/installation-using-ansible.md:35]: Golang version is going to be updated to v1.13 (exact version to be provided shortly)

File: content/try-opensds/installation-using-ansible.md [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+```bash
+go version
+```
+<br />It should be go1.12.1 linux/amd64
+
+### Download SODA installer code
+```bash
+git clone https://github.com/sodafoundation/installer.git
+cd installer/ansible
+# Checkout the latest stable release. Current stable release: stable/elba. If you want to get the master branch of all components, you can skip this step. (<span style="color:red"> Attn:</span> Master may not be stable or tested fully)
```
Comment (anvithks) [inline content/try-opensds/installation-using-ansible.md:51]: `(<span style="color:red"> Attn:</span> Master may not be stable or tested fully)`
This does not render as html even in the wiki.
You can use the alert box for this on our docs website.
Use the following code:
```
{{% notice warning %}}
Master may not be stable or tested fully
{{% /notice %}}
```

But this notice will have to be outside the code block.

File: content/try-opensds/installation-using-ansible.md [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+weight: 30
+---
+This document describes how to install SODA projects local cluster, including Hotpot (API, Controller, Dock), Gelato, Telemetry, Orchestration, and Dashboard components.
+
+`Hereafter, Hotpot refers to API, Controller and Dock projects`
+
+### Pre-config (Ubuntu 16.04)
+All the installation work is tested on `Ubuntu 16.04`, please make sure you have installed the right one. Also `root` user is REQUIRED before the installation work starts.
+
+<br /> Install following packages:
```
Comment (anvithks) [inline content/try-opensds/installation-using-ansible.md:13]: nit: use #### for subheadings  

### Pre-config (Ubuntu 16.04)
....
#### Install following packages
...
#### Install Docker
....
#### Install docker-compose

File: content/try-opensds/installation-using-ansible.md [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+<br />
+
+This step is needed to upgrade ansible to version 2.4.2 which is required for the "include_tasks" ansible command.
+<br />
+
+```bash
+chmod +x ./install_ansible.sh && ./install_ansible.sh
+ansible --version # Ansible version 2.4.x is required.
+```
+<br />Configure soda hotpot install variables:
```
Comment (anvithks) [inline content/try-opensds/installation-using-ansible.md:65]: soda -> SODA

File: content/try-opensds/installation-using-ansible.md [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+
+Firstly you need to modify host_ip in `group_vars/common.yml`, and you can specify which project (hotpot or gelato) to be deployed:
```
Comment (anvithks) [inline content/try-opensds/installation-using-ansible.md:67]: Sentence structure.
**Firstly you need to modify host_ip...**  -> **Modify host_ip and change it to the actual machine IP of the host. You can specify which projects to deploy; only hotpot, only gelato or all**
The project names are going to change. Please change later when the names are finalised. For now we use hotpot and gelato.

File: content/try-opensds/installation-using-ansible.md [hunk 8]
```diff
@@ -0,0 +1,278 @@
...
+```bash
+ansible-playbook site.yml -i local.hosts
+# You can use the -vvv or -vv option to enable verbose display and debug mode.
+[verbosity level: -vv < -vvv]
+ansible-playbook site.yml -i local.hosts -vvv
+```
+# How to test SODA projects cluster
+<br />SODA projects CLI
+
+Firstly configure SODA projects CLI tool:
```
Comment (anvithks) [inline content/try-opensds/installation-using-ansible.md:181]: Firstly configure... -> Configure SODA Projects...

File: content/try-opensds/installation-using-ansible.md [hunk 9]
```diff
@@ -0,0 +1,278 @@
...
+osdsctl fileshare list
+```
+Delete the Fileshare
+```bash
+osdsctl fileshare delete <fileshare id>
+```
+### SODA Dashboard UI
+<br />
+
+SODA Dashboard UI dashboard is available at `http://{your_host_ip}:8088`, please login the dashboard using the default admin credentials: `admin/opensds@123.` Create `tenant`, `user`, and `profiles` as admin. Multi-Cloud service is also supported by dashboard.
```
Comment (anvithks) [inline content/try-opensds/installation-using-ansible.md:235]: SODA Dashboard UI dashboard  -> SODA Dashboard UI is available

**Comment by anvithks** [comment]
> Thank you for reviewing.. I will work on the issue and update soon. Also! If we can add Doc again for Ubuntu 18 and other versions. When official support will arrive I don't think we are going to need steps for 16.04

There may be some steps that may be different but most of the steps would remain same.

**Comment by skdwriting** [comment]
@ravisaxena23 , could you please confirm the review fixes and let me know. Thanks a lot for your contribution.

**Comment by skdwriting** [comment]
@ravisaxena23 please update the comments reply. So we can merge before the upcoming release.

File: content/try-opensds/installation-using-ansible.md [hunk 10]
```diff
@@ -0,0 +1,278 @@
...
+tar -C /usr/local -xzf go1.12.1.linux-amd64.tar.gz
+echo 'export PATH=$PATH:/usr/local/go/bin' >> /etc/profile
+echo 'export GOPATH=$HOME/gopath' >> /etc/profile
+source /etc/profile
+```
+`Check if golang exists`
+```bash
+go version
+```
+<br />It should be go1.12.1 linux/amd64
```
[the remark this answers is not in the document]
Comment (ravisaxena23) [reply content/try-opensds/installation-using-ansible.md:45]: @skdwriting Im wating for this reply

**Comment by ravisaxena23** [comment]
> Thank you @ravisaxena23 .
> This is a good first contribution.
> I have made some suggestions and nits.
> Our docs website is built using Hugo / Netlify and the Learn theme for hugo.
> The theme offers some styling elements and shortcodes. You can find the documentation for the theme [here](https://learn.netlify.app/en/)

thank you sir

File: content/try-opensds/installation-using-ansible.md (hunk 3 above)
Comment (ravisaxena23) [reply content/try-opensds/installation-using-ansible.md:35]: @skdwriting waiting for this

File: content/try-opensds/installation.md (hunk 1 above)
Comment (ravisaxena23) [reply content/try-opensds/installation.md:15]: I have added link to new installation readme. I have checked and link works in my code editor.

**Pull request closed without merging**